bin/.gitignore

```diff
@@ -1,2 +1,2 @@
 cache
-
+flutter.lock
```

Comment:
Talked in person. The other lockfile is also never checked in. We decided to move this lock file into the cache dir, though.